wallace/command_line.py

```diff
@@ -344,6 +344,11 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1):
 
     database_size = config.get('Database Parameters', 'database_size')
 
+    try:
+        whimsical = config.get('Easter eggs', 'whimsical') == "true"
```

Comment:
Instead of `=== "true"`, ConfigParser has a function `getboolean` which does the conversion for you.